docs/src/usage/output-formats.md

```
@@ -0,0 +1,309 @@
+## CSV Format
+
+**Comma-Separated Values** - Industry standard tabular format.
+
+### Specifications
+
+- **Standard**: RFC4180-compliant
+- **Quoting**: `QuoteStyle::Necessary` (only when required)
+- **Line Endings**: CRLF (`\r\n`)
+- **NULL Handling**: Empty strings
+- **Encoding**: UTF-8
+
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)